nixos/tests/networking/router.nix

```
@@ -0,0 +1,82 @@
+{ networkd }: {config, pkgs, ... }:
```

```suggestion
{ networkd }: { config, pkgs, ... }:
```

inconsistent whitespace :V

---

fixed